lms/static/scripts/frontend_apps/utils/api.js

```diff
@@ -126,3 +131,23 @@ export async function apiCall(options) {
 
   return resultJSON;
 }
+
+/**
+ * Template tag that formats a URL path, ensuring interpolated strings are
```

Comment:
```suggestion
 * Template tag function that formats a URL path, ensuring interpolated strings are
```

Pedantic, but sometimes the difference between "tagged template [literals]" and the associated "tag function" is confusing.